stable/kommander/requirements.lock

```diff
@@ -4,10 +4,10 @@ dependencies:
   version: 0.1.4
 - name: kommander-cluster-lifecycle
   repository: https://mesosphere.github.io/kommander-cluster-lifecycle/charts
-  version: v0.4.6
```

Comment:
Was the "v" a mistake or are we changing the format?

Reply:
both works, I think its depedning on different helm versions? its a lockfile - wasnt a manual change.